packages/framework-core/test/decorators/command.test.ts

```diff
@@ -32,7 +32,9 @@ describe('the `Command` decorator', () => {
       expect(commandMetadata).to.be.an('object')
       expect(commandMetadata.class).to.equal(PostComment)
       expect(commandMetadata.properties[0].name).to.equal('comment')
+      console.log('WAT')
       expect(commandMetadata.properties[0].typeInfo.name).to.equal('string')
+      console.log('WAT')
```

Comment:
These seem to be debugging leftovers, I guess you can remove them now that the PR is ready for merge

```suggestion
      expect(commandMetadata.properties[0].typeInfo.name).to.equal('string')
```

Reply:
Thanks for catching those! You can see my despair in these lines lol